src/esp/sim/SimulatorConfiguration.h

```diff
@@ -5,11 +5,15 @@
 #ifndef ESP_SIM_SIMULATORCONFIGURATION_H_
 #define ESP_SIM_SIMULATORCONFIGURATION_H_
 
+#include <Corrade/Containers/Optional.h>
 #include <string>
 
 #include "esp/core/Esp.h"
+#include "esp/nav/PathFinder.h"
```

Comment:
This should probably be forward-declared as this class is widely used.

Reply:
Yeah, I thought so too. However, the compiler does not like the forward declaration here, so I needed the full the include.